### PR TITLE
Uninstall message box

### PR DESCRIPTION
**Pull request by Odelpasso**
Hello,

I add a Message box to confirm if an user wants to uninstall a game or not.

### CODE REVIEW

**Comment by sharkwouter** [comment]
Thanks for the work! It looks pretty good, but I won't be able to test it until tonight

File: minigalaxy/window/gametile.py [hunk 1]
```diff
@@ -69,11 +69,23 @@ def on_button_click(self, widget) -> None:
...
-        self.menu_button.hide()
-        self.button.set_sensitive(False)
-        self.button.set_label(_("uninstalling.."))
-        uninstall_thread = threading.Thread(target=self.__uninstall_game)
-        uninstall_thread.start()
+        message_dialog = Gtk.MessageDialog(parent=self.parent,
+                                           flags=Gtk.DialogFlags.MODAL,
+                                           message_type=Gtk.MessageType.WARNING,
+                                           buttons=Gtk.ButtonsType.OK_CANCEL,
+                                           message_format="Are you sure to uninstall this game ?")
```
Comment (sharkwouter) [inline minigalaxy/window/gametile.py]: Could you replace this game with the name of the game?

Comment (sharkwouter) [reply minigalaxy/window/gametile.py]: Could you also make the string translatable? I've added instructions on how to do that here: https://github.com/sharkwouter/minigalaxy/wiki/Developer-information#making-strings-translatable

**Comment by Odelpasso** [comment]
I think it is good boss.

**Pull request merged**